lua/entities/gmod_wire_expression2/core/sound.lua

```diff
@@ -178,23 +295,49 @@ e2function void soundPitch( index, pitch )
 	local sound = getSound( self, index )
 	if not sound then return end
 	
-	sound:ChangePitch( math.Clamp( pitch, 0, 255 ), 0 )
+	sound:ChangePitch( math.Clamp( pitch, 0, 300 ) / 100, 0 )
 end
 
 e2function void soundPitch( index, pitch, fadetime )
 	local sound = getSound( self, index )
 	if not sound then return end
 	
-	sound:ChangePitch( math.Clamp( pitch, 0, 255 ), math.abs( fadetime ) )
+	sound:ChangePitch( math.Clamp( pitch, 0, 300 ) / 100, math.abs( fadetime ) )
```

Comment:
Actually if its still a multiplier, disregard keeping it consistent to the same range. math.Clamp( pitch, 0, 400 ) / 100 sounds fine.